src/gd_filename.c

```diff
@@ -37,7 +37,7 @@ static void BGD_STDCALL writebmp(gdImagePtr im, FILE *out) {
 }/* writejpeg*/
 
 
-enum FType {UNKNOWN, PNG, JPG, GIF, TIFF, GD, GD2, WEBP};
+enum FType {UNKNOWN, PNG, JPG, GIF, TIFF, GD, GD2, WEBP, AVIF};
```

Comment:
please rebase onto the latest master ... i've deleted this enum as it's unused

Reply:
Hmm, I thought I'd done this, but it doesn't look like I did it right. I'll re-rebase